light_curves/code/sample_lc.py

```diff
@@ -0,0 +1,145 @@
+import sys
+import time
+import warnings
+from math import ceil
+from multiprocessing import Pool
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+import matplotlib.pyplot as plt
+import numpy as np
+from astropy.table import Table
+from scipy import stats
+import pandas as pd
+from panstarrs import panstarrs_get_lightcurves
+from gaia_functions import Gaia_get_lightcurve
+from HCV_functions import HCV_get_lightcurves
+from icecube_functions import icecube_get_lightcurve
+from sample_selection import get_lamassa_sample, get_macleod16_sample, get_ruan_sample, get_macleod19_sample, get_sheng_sample, get_green_sample, get_lyu_sample, get_lopeznavas_sample, get_hon_sample, get_yang_sample,get_SDSS_sample, get_paper_sample, clean_sample,nonunique_sample,TDE_id2coord
+from data_structures import MultiIndexDFObject
+from heasarc_functions import HEASARC_get_lightcurves
+from TESS_Kepler_functions import TESS_Kepler_get_lightcurves
+from WISE_functions import WISE_get_lightcurves
+from ztf_functions import ZTF_get_lightcurve
+from ML_utils import unify_lc, stat_bands, autopct_format, combine_bands,\
+mean_fractional_variation, normalize_mean_objects, normalize_max_objects, \
+normalize_clipmax_objects, shuffle_datalabel, dtw_distance, stretch_small_values_arctan
+
```

Comment:
```suggestion


# make sure the data and output directories exist so that we can write to them
os.makedirs("../data", exist_ok=True)
os.makedirs("../output", exist_ok=True)

```